server/users.js

```diff
@@ -21,31 +19,116 @@ module.exports = require('express').Router()
     // forbidden('listing users is not allowed'),
     (req, res, next) =>
     User.findAll()
-    .then(users => res.json(users))
+    .then(users => res.status(200).json(users))
+    .catch(next))
+  // query to see non-admin users
+  .get('/?isAdmin="false"', (req, res, next) =>
+    User.findAll({where: {isAdmin: req.query.isAdmin}})
+    .then(users => res.status(200).json(users))
+    .catch(next))
+  // query to see if user is admin
+  .get('/?isAdmin="true"', (req, res, next) =>
+    User.findAll({where: {isAdmin: req.query.isAdmin}})
+    .then(users => res.status(200).json(users))
+    .catch(next))
+  // admin or actual user can view
+  .get('/:id', (req, res, next) =>
+    User.findById(req.params.id, {
+      include: [Review]
+    })
+    .then(user => res.json(user))
+    .catch(next))
+  // cant create a new user with admin status
+  .post('/', (req, res, next) =>
+      User.create(req.body)
+      .then(user => res.status(201).json(user))
+      .catch(next))
+  .put('/:id', (req, res, next) =>
```

Comment:
looks like anyone can update a user right now. Users should be update their own accounts. Admins should be able to update everyone's accounts.